testbed/tests/widgets/properties.py

```diff
@@ -15,30 +15,6 @@ async def test_text(widget, probe):
         assert probe.text == text
 
 
-async def test_text_empty(widget, probe):
-    "The text displayed on a widget can be empty"
-    # Set the text to the empty string
-    widget.text = ""
```

Comment:
Empty strings are now covered by `test_text` above, and conversion of non-string objects is covered by test_button.py in the core.